src/migrations/20230802092725-add-last-seen-column-to-feature-environments.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql(
+        `
+          ALTER TABLE feature_environments ADD COLUMN IF NOT EXISTS last_seen_at timestamp with time zone;
+
+          DROP VIEW features_view;
```

Comment:
maybe  CREATE OR REPLACE VIEW ?

Reply:
This was necessary to avoid ALTER VieW RENAME COLUMN